public/css/timeline.less

```diff
@@ -28,6 +28,20 @@
         margin-left: .5em;
         .text-ellipsis();
       }
+
+      > a {
+        display: flex;
+        width: 100%;
+        align-items: baseline;
+        margin-left: .5em;
+
+        span {
+          width: 0;
+          flex: 1 0 auto;
+          margin-right: 0.25em;
+          .text-ellipsis();
+        }
```

Comment:
The `> span` rule above was the basis for this I suppose. Why is it still there though? The timeline now always renders a link in the sidebar.